app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -409,7 +409,7 @@ class AdvancedFilter extends React.Component {
     if (value === 'within') {
       defaultValue = {
         start: moment()
-          .add(-72, 'hours')
+          .subtract(3, 'days')
```

Comment:
Why was this changed?

Reply:
to be easier to read, why add negative hours when you can just subtract them

Reply:
we use `.subtract` in other places when going back in time

Reply:
Makes sense - thanks!